Assets/_BForBoss/_LevelDesign/Materials/HeightMap/M_BlueGreeb.mat

```diff
@@ -0,0 +1,143 @@
+%YAML 1.1
```

Comment:
What is greeb?

Reply:
It stands for Greeble which is the patterns that the materials is based on the height map. Could be called anything though really, that just made the most sense to me.